Kickstarter-iOS/Library/RemoteConfigClient.swift

```diff
@@ -0,0 +1,62 @@
+import Firebase
```

Comment:
So I think we don't need this file.

Here's my thinking: 

The pattern we followed for `OptimizelyClient` is to create it in `AppDelegate` and then conform a protocol for functions we need from it. It was `OptimizelyClientType`.

Then all we do is pass around that `OptimizelyClientType` instance via `AppEnvironment`. 

This is why the real `OptimizelyClient` could easily be replaced with the mock version when we removed the library.

With the new `RemoteConfigClientType` we already conform it to whatever methods we get access to in `RemoteConfig.remoteConfig()`and pass around the `RemoteConfigClientType`, so there is no need for this wrapper class around `RemoteConfig.remoteConfig()` singleton. Just like there was no need for a wrapper around `OptimizelyClient` from `Optimizely` framework.

It's also easier to mock - for that matter add a `MockRemoteConfigClient` in the same way we had `MockOptimizelyClientType`.